packages/opentelemetry-node/lib/elastic-node-sdk.js

```diff
@@ -72,8 +87,23 @@ class ElasticNodeSDK extends NodeSDK {
         defaultConfig.instrumentations =
             opts.instrumentations || getInstrumentations();
 
-        // Default logs exporter.
-        // TODO: handle other protocols per OTEL_ exporter envvars (or get core NodeSDK to do it). Currently hardcoding to http/proto
+        // Default logs exporter based on environment. Using `http/proto` as default protocol
+        const logsExporters = {
+            grpc: OTLPLogExporterGrpc,
+            'http/json': OTLPLogExporterHttp,
+            'http/protobuf': OTLPLogExporterProto,
```

Comment:
Do you think it would be worthwhile to *lazily* import these exporters if a custom protocol config is given? In the common case, we'll be loading deps that don't get used. Probably not a big deal as our default of 'http/protobuf' means we'll be loading the heavier deps anyway.

Or perhaps getting into lazy loading of deps could cause other issues down the road: e.g. if/when we support working when bundled.

Reply:
If config sets to a different protocol than the default we won't get the heaviest dependency if we require the exporter at runtime. That's a good point :)

I guess we will have issues in the future with bundlers but it's a problem quite difficult to plan ahead. IIUC how bundlers work they scan code to check which dependencies are/will be used at runtime to inline them with the module code. And in this case it depends on some env vars which is completely out of sight of the bundler.

I'm going to follow your suggestion. We can discuss later the bundler compatibility